notebooks/official/tabnet/tabnet_vertex_tutorial.ipynb

```diff
@@ -0,0 +1,972 @@
+{
```

Comment:
<p>Please look at the <a href="https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb" target="_blank">notebook template</a> for ideas on anything else that needs to be deleted.</p><p> </p><p>In particular, the bucket should be deleted (ONLY when in testing mode -- the if statement is in the notebook template). This prevents buckets from being created over and over again every time the CI process runs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/375/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>yes, I add for removing the bucket.</p><p>In this tutorial, we focus on training so deleting the JOB_DIR and BQ is sufficient.</p>